plugins/nid-team/rebase/rebase_manager.py

```
@@ -0,0 +1,767 @@
+    def get_github_repo_name(self):
+        """Extract repo name (e.g. openshift/coredns) from gh or git remotes."""
+        try:
+            return self.run_cmd(["gh", "repo", "view", "--json", "nameWithOwner", "-q", ".nameWithOwner"])
+        except Exception:
+            # Fallback to git remote parsing
+            for remote in ["openshift", "upstream", "origin"]:
+                try:
+                    remote_v = self.run_cmd(["git", "remote", "get-url", remote])
+                    break
+                except Exception:
+                    continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)